infra/concourse/build/Dockerfile.kitchen-terraform

```diff
@@ -35,6 +35,12 @@ SHELL ["/bin/bash", "-c"]
 
 ENV APP_BASE_DIR="/cft"
 
+RUN cd /tmp && \
+    wget https://releases.hashicorp.com/packer/1.3.3/packer_1.3.3_linux_amd64.zip && \
```

Comment:
Let's use the latest release of Packer, 1.4.1.

Reply:
Ok, I'll change it, thank you!

Reply:
Done